src/features/vars-schema/env-vars.ts

```diff
@@ -73,6 +77,14 @@ export const SUPERVISOR_CONFIG_VAR_PROPERTIES: {
 		description: 'Enable / Disable logs from being sent to balena API',
 		default: 'true',
 	},
+	RESIN_SUPERVISOR_REPORT_INTERVAL: {
+		type: 'integer',
+		description:
+			'Define how frequently the device reports its current state to the cloud in milliseconds',
```

Comment:
Shouldn't we also add the classic `Supported by supervisor versions >= v13.?.?.` at the end like we do in other places for new config vars?